pwnlib/crypto/__init__.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+
+# load cipher suites
+from pwnlib.crypto.ciphers.atbash import CipherAtBash
+from pwnlib.crypto.ciphers.bacon import CipherBacon
+from pwnlib.crypto.ciphers.base64 import CipherBase64
+from pwnlib.crypto.ciphers.binary import CipherBinary
+from pwnlib.crypto.ciphers.caesar import CipherCaesar
+from pwnlib.crypto.ciphers.decimal import CipherDecimal
+from pwnlib.crypto.ciphers.hex import CipherHex
+from pwnlib.crypto.ciphers.morse import CipherMorse
+from pwnlib.crypto.ciphers.reverse import CipherReverse
+from pwnlib.crypto.ciphers.rot13 import CipherRot13
+from pwnlib.crypto.ciphers.transposition import CipherTransposition
+from pwnlib.crypto.ciphers.vigenere import CipherVigenere
+from pwnlib.crypto.ciphers.xor import CipherXor
+
+class Crypto():
```

Comment:
You should be able to use the __module__ hack seen elsewhere in Pwntools to make this "automagic"

Reply:
Im not sure what you mean with this, can you give me an example?

Reply:
Can you please also tell me, how to follow your suggestions about making routines top-level as you told here:

Edit: It would also be nice if all of these routines were exposed as top-level routines, e.g. crypto.rot13 rather than CryptoRot13. By convention, pretty much everything in Pwntools is lower-case.

I could rename the classes, but im not sure how to do this the way you want it to, that it fits.